crates/compiler/builtins/benchmark-dec.zig

```diff
@@ -2,8 +2,8 @@ const std = @import("std");
 const time = std.time;
 const Timer = time.Timer;
 
-const RocStr = @import("../src/str.zig").RocStr;
-const RocDec = @import("../src/dec.zig").RocDec;
+const RocStr = @import("./bitcode/src/str.zig").RocStr;
+const RocDec = @import("./bitcode/src/dec.zig").RocDec;
```

Comment:
We moved this file, because zig 0.13.0 no longer allows relative imports from a parent/ancestor.